index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Grow a tree</title>
+	<link rel="stylesheet" href="">
+</head>
+<body>
+	<div><label>How many branches will the tree have?<input type="text" id="branches"></label></div>
```

Comment:
While single lining is OK when you're only one tag deep, with multiple tags like this it's better to split them up so you can see what's going on easier via indentation.

Also, putting the input inside of the label tag isn't really how labels for input fields work, best practice is to add a ```for``` attribute to the label tag, tied to the input's ID, i.e.:
```
<label for='foo'>This is foo</label>
<input id='foo' type='text>
```
This ties the label to the input in a way screen readers etc. can understand.